src/test/resources/data/500Eori.json

```diff
@@ -0,0 +1,5 @@
476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179","XI270463476179"]
```

Comment:
Please can you run these through a formatter.

Reply:
Formatted the json.